tests/test_all_rules.py

```diff
@@ -171,6 +171,22 @@ def test_max_signals_note(self):
                               f'Expected: {max_signal_standard_setup}\n\n'
                               f'Actual: {rule.contents.data.setup}')
 
+    def test_from_filed_value(self):
+        """ Add "from" Field Validation for All Rules"""
+        failures = []
+        valid_format = re.compile(r'^now-\d+(y|M|w|d|H|h|m|s)$')
```

Comment:
Im not sure this covers all the use cases in the documentation.  e.g `now+1h`

You could try something a bit more complicated like `r'^(now|\d{4}\.\d{2}\.\d{2})\|\|[-+]\d+[yMwdhHmms](/[yMwdhHmms])?$'` but you'll need to test it.

Reply:
The `now+1h` was mostly for the kibana events was my understanding. From the rule perspective do we see alerts firing for timeline `now+1h` ? Hence the focus was on formats valid for rule development.

Reply:
In that case, it may make more sense to document in the summary which ones we're not considering and why. Based on the description, it sounds like we intended to provide full support.

Reply:
Updated Summary an opted for a slightly better regex